creator-node/scripts/run-tests.sh

```diff
@@ -57,6 +57,7 @@ ARG1=${@:$OPTIND:1}
 
 if [ "${ARG1}" == "standalone_creator" ]; then
   export redisPort=4377
+  export redisHost=127.0.0.1
```

Comment:
why's this change required?

Reply:
I don't exactly remember why but I think it has to do with how the CN would fail to bring up the queues bc the redisHost was not defined